diff --git a/source-builder/config/gcc-common-1.cfg b/source-builder/config/gcc-common-1.cfg
index dd0cf4c..663fe16 100644
--- a/source-builder/config/gcc-common-1.cfg
+++ b/source-builder/config/gcc-common-1.cfg
@@ -147,6 +147,7 @@ BuildRoot: %{_tmppath}/%{name}-root-%(%{__id_u} -n)
     --disable-win32-registry \
     --enable-version-specific-runtime-libs \
     --disable-lto \
+    --enable-target-optspace \
     --enable-newlib-io-c99-formats \
     %{?with_iconv:--enable-newlib-iconv} \
     %{?with_iconv:--enable-newlib-iconv-encodings=%{_newlib_iconv_encodings}} \
